src/Itinerary.js

```diff
@@ -0,0 +1,5 @@
+function Itinerary(ports){
```

Comment:
Could these all be created as classes utilising constructors instead?
Eg:
```
class Itinerary {
    constructor({ ports }) {
        this.ports = ports;
    }
}
```

In any case, adding a space before the opening function bracket on line would be good